cmd/traffic/cmd/manager/internal/mutator/watcher.go

```diff
@@ -205,6 +245,15 @@ func (c *configWatcher) Store(ctx context.Context, ac *agentconfig.Sidecar, upda
 			return fmt.Errorf("unable to get ConfigMap %s: %w", agentconfig.ConfigMap, err)
 		}
 		create = true
+	} else {
+		// Ensure that we're not about to overwrite a manually added config entry
+		if currentYml, ok := cm.Data[ac.AgentName]; ok {
+			var currAc agentconfig.Sidecar
+			if err = decode(currentYml, &currAc); err == nil && currAc.Manual {
+				dlog.Warn(ctx, "avoided an attempt to overwrite manually added config entry for %s.%s", ac.AgentName, ns)
```

Comment:
I think this should be warnf since you have a formatting directive at the end there